frontend/apps/filemanager/filemanager.lua

```diff
@@ -477,8 +478,8 @@ function FileManager:onClose()
     return true
 end
 
-function FileManager:onRefresh()
-    self.file_chooser:refreshPath()
+function FileManager:onRefresh(value)
+    self.file_chooser:refreshPath(value)
```

Comment:
I would replace `value` with `update_perpage` as it is named in the function you call, so we know here what value means.
But I'm not sure any value is needed, see next comment.